### DISCUSSION TITLE
Suggestion:please make own keyboard

### DISCUSSION THREAD

**Discussion by disk123s** [AppGround-io/bluetooth-keyboard-and-mouse-support#185] [Ideas]
Please make your own keyboard. The response is slow because it is re-sent after entering the key. I want to use my tablet as a Quarty keyboard in the library! thanks

**Comment by zorazorr** [comment]
Maybe try an other keyboard app.
Ii can recommend "Hacker's Keyboard" in combination with "Serverless Bluetooth Keyboard & Mouse":
https://play.google.com/store/apps/details?id=org.pocketworkstation.pckeyboard
